scripts/push-image.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+set -u
+
+env >> BASH_ENV
+cat BASH_ENV | while read line; do
+	export $line
+done
+if [ -z "${CIRCLE_TAG:-}" ]; then
```

Comment:
Assumes usage of `CircleCI` need to be changed accordingly with the value of the preffered CI or make it variable to be passed at runtime.